index.html

```diff
@@ -121,22 +121,22 @@ <h2>Introduction</h2>
       <em>This section is non-normative.</em>
 
       <p>
-        Welcome dear implementer of Verifiable Credentials! We understand that reading
-        and nagivating technical specifications can be confusing at times. For this reason,
-        we would like to offer you a few resources to get you started.
+Reading and navigating technical specifications can be confusing. For this
+reason we have created this guide, which provides some examples and resources
+for implementing protocols which make use of <a>verifiable credentials</a>.
       </p>
       <p>
-        First, you should probably become familiarized with the official
-        <a href="https://www.w3.org/TR/verifiable-claims-use-cases/">Use Cases document</a>,
-        which offers a concise but limited collection of use cases readers should review before
-        undestanding how Verifiable Claims can best help them.
+It may be useful to become familiarized with the official
+<a href="https://www.w3.org/TR/verifiable-claims-use-cases/">Use Cases document</a>,
+which offers a concise collection of examples of <a>Verifiable Credentials</a>
+as they may appear in everyday life, and how they may be used.
       </p>
       <p>
-        The <a href="https://w3c.github.io/vc-data-model/">data model specification</a> is the
-        most important result of two years of hard work done by the Verifiable Claims Working Group.
-        However, the data model itself is not sufficient if you plan to use Verifiable Claims today.
-        Additional works needs to be done on identifiers as well as on proofs, as you will be able to
-        find out in the sections below.
+The <a href="https://w3c.github.io/vc-data-model/">data model specification</a>
+contains the technical details about <a>verifiable credentials</a>. However, the
+data model itself does not specify any protocols for using <a>verifiable
+credentials</a>. Additionally, these protocols may depend on identifiers and
+proof formats which themselves require more specification.
```

Comment:
```
data model itself does not specify any protocols for using <a>verifiable 
credentials</a>. Additionally, these protocols may depend on identifiers and
proof formats which themselves require more specification.
```
->
```
data model specification does not specify any <em>protocols</em> for using
<a>verifiable credentials</a>, nor any <em>proof formats</em> or additional
<em>identifiers</em> upon which such protocols may depend.
```

Reply:
Thanks, fixed in [aa459c1](https://github.com/w3c/vc-imp-guide/pull/26/commits/aa459c1497ccdfa8682ca2ed8138a7b747340ff4)